dev/com.ibm.ws.webcontainer.servlet.3.1.factories/bnd.bnd

```diff
@@ -108,3 +108,5 @@ instrument.disabled: true
     com.ibm.ws.javaee.version;version=latest,\
     com.ibm.ws.artifact;version=latest,\
     com.ibm.ws.artifact.overlay;version=latest
+    
+jakartaeeMe: true
```

Comment:
I don't see where this transformed bundle is used in any of the committed sources.  Is this change necessary?